composer/datasets/in_context_learning_evaluation.py

```diff
@@ -1288,15 +1293,27 @@ def __init__(
             'generation_kwargs': {
                 'pad_token_id': self.pad_tok_id,
                 'num_beams': 1,  # single beam
-                'num_return_sequences': generations_per_sample,
```

Comment:
You should update the documentation for `InContextLearningCodeEvalDataset` to remove the line about putting `num_return_sequences` in `generations_kwargs`. Specifically, you should remove line 1225 of the file:
```
- num_return_sequences: Value passed for 'generations_per_sample', how many generations per prompt
```
This will make things a lot clearer

Reply:
good catch, thanks